### PR TITLE
Remove BackTo

### LINKED ISSUE #7221: BackTo element in the new design
## Bug description

We have this "BackTo" component. It's used in a lot of places:

- In the actions table when going to an insight (screencast above)

- Clicking into an action in the taxonomic filter

- Cohorts table

- Events table

- The persons table when going into a person

- The persons table under cohorts

- The persons table in the retention insights's line graph (now that's an obscure place!)

- Few places in the old `scenes/sessions`
- Few other action dropdowns (`ActionInfo`)

## Usage

Excluding "Back to recordings/sessions" clicks from within recordings, a "Back to" link is clicked on average by [5% of users](https://app.posthog.com/insights?insight=TRENDS&interval=day&display=ActionsLineGraph&actions=%5B%7B%22id%22%3A12253%2C%22name%22%3A%22Clicked%20%5C%22Back%20To%5C%22%22%2C%22type%22%3A%22actions%22%2C%22order%22%3A0%2C%22math%22%3A%22dau%22%7D%5D&events=%5B%7B%22id%22%3A%22%24pageview%22%2C%22type%22%3A%22events%22%2C%22order%22%3A1%2C%22name%22%3A%22%24pageview%22%2C%22math%22%3A%22dau%22%7D%5D&new_entity=%5B%5D&properties=%5B%7B%22key%22%3A%22%24current_url%22%2C%22value%22%3A%22https%3A%2F%2Fapp.posthog.com%2F%22%2C%22operator%22%3A%22icontains%22%2C%22type%22%3A%22event%22%7D%5D&formula=(A%20%2F%20B)%20*%20100) on cloud:

image

## Questions / issues

- Do we need this link?
- Normally the browser's "back" button is made to perform this function. We should strive to have that button working well on all pages. It usually does, not like in one of the screencasts above (link to old sessions page from the cohorts table).
- If we need this, is the element correctly designed/implemented/positioned now?
- Currently this BackTo component shows sooner than the next scene, leading to weird effects. This should be fixed if we decide to keep it.

Any strong feelings for keeping or removing? @clarkus @Twixes @paolodamico @timgl ?

### PR DESCRIPTION
**Pull request by mariusandra**
## Changes

- Resolves https://github.com/PostHog/posthog/issues/7221
- Fixes/modernises a few urls in the process, except for insight URLs which will get their own PR

## How did you test this code?

- Clicked around in the interface, saw that the links still worked and no component popped up.

### CODE REVIEW

**Review by clarkus — APPROVED** [review]

File: frontend/src/scenes/insights/ActionFilter/ActionFilterRow/ActionFilterDropdown.tsx [hunk 1]
```diff
@@ -12,13 +13,7 @@ export function ActionInfo({ item }: RenderInfoProps): JSX.Element {
...
         <>
             <AimOutlined /> Actions
-            <Link
-                to={`/action/${item.id}#backTo=Insights&backToURL=${encodeURIComponent(
-                    window.location.pathname + window.location.search
-                )}`}
-                style={{ float: 'right' }}
-                tabIndex={-1}
-            >
+            <Link to={item.id ? urls.action(item.id) : urls.createAction()} style={{ float: 'right' }} tabIndex={-1}>
```
Comment (Twixes) [inline frontend/src/scenes/insights/ActionFilter/ActionFilterRow/ActionFilterDropdown.tsx:16]: Shouldn't we just say that there's no such action if there's no ID?

**Review by Twixes — COMMENTED** [review]
Commendable

File: frontend/src/scenes/insights/ActionFilter/ActionFilterRow/ActionFilterDropdown.tsx (hunk 1 above)
Comment (mariusandra) [reply frontend/src/scenes/insights/ActionFilter/ActionFilterRow/ActionFilterDropdown.tsx]: Indeed. Fixed by just removing the link if there's no ID. I can't be sure if this is called with new actions anywhere now or in the future, so this seemed like the shortest and safest fix. Same with cohorts below as well.

**Comment by mariusandra** [comment]
Fixed the feedback, going to merge, as this also contains a fix to the e2e tests and touches multiple files.

**Pull request merged**